src/main/java/modtrekt/ui/TaskCard.java

```diff
@@ -1,16 +1,14 @@
 package modtrekt.ui;
 
-import java.util.Comparator;
-
 import javafx.fxml.FXML;
 import javafx.scene.control.Label;
 import javafx.scene.layout.FlowPane;
 import javafx.scene.layout.HBox;
 import javafx.scene.layout.Region;
-import modtrekt.model.person.Person;
+import modtrekt.model.module.Module;
 
 /**
- * An UI component that displays information of a {@code Person}.
+ * An UI component that displays information of a {@code Module}.
  */
 public class TaskCard extends UiPart<Region> {
```

Comment:
Perhaps could add another class representing the Module card and Module panel, but I think could leave it to the people doing up the UI as well, but currently, this may intefere with the Task section when merging

Reply:
I've already created `ModuleCard` and `ModuleListPanel` in the `ui/modules` package in my PR - should this class be deleted?